package.json

```diff
@@ -4,11 +4,13 @@
   "description": "a website for campgrounds",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node app.js"
   },
   "author": "Rachamim Dabbah",
   "license": "ISC",
   "dependencies": {
+    "body-parser": "^1.17.2",
```

Comment:
I needed to add this b/c it was missing and the app couldn't boot up. It technically doesn't have anything to do with the start script.